### ISSUE TITLE
no-op in copyInputStyles when a component is mounted

### ISSUE THREAD

**Issue by jianliangzhao** [JedWatson/react-input-autosize#92]
Here is the check in `copyInputStyles`:

```
	copyInputStyles () {
		if (this.mounted || !window.getComputedStyle) {
			return;
		}
```

The `copyInputStyles` is called right after `this.mounted` is set to `true` in `compomentDidMount`. Shouldn't the condition be `!this.mounted` instead?

```
        componentDidMount: function componentDidMount() {
                this.mounted = true;
                this.copyInputStyles();
                this.updateInputWidth();
        },
```

**Comment by jianliangzhao** [comment]
Ok, someone had a fix for this issue already:

`https://github.com/JedWatson/react-input-autosize/pull/91`

**Comment by JedWatson** [comment]
Fixed in 1.2.0+
